Documentation/design/resource_dep.dot

```diff
@@ -196,11 +204,10 @@ strict digraph resource {
 
   kubeconfig_kubelet -> manifests;
   kubeconfig_admin -> manifests;
-  kubeconfig_admin -> cluster;
 
   bootstrap_ignition -> ignition_configs;
   master_ignition -> ignition_configs;
   worker_ignition -> ignition_configs;
 
-  terraform_state -> cluster;  
+  openshift_cluster -> cluster;
 }
```

Comment:
I think we should reserve the cluster target for if/when the installer invokes terraform on behalf of the user. Maybe call this target cluster-inputs?